### PR TITLE
Add remove.bg extension.

### PR DESCRIPTION
**Pull request by maantje**
## Description

The remove.bg extension adds the ability to remove a background from an image using the [https://www.remove.bg/](https://www.remove.bg/) API with the [remove.bg API wrapper for Node.js](https://www.npmjs.com/package/remove.bg).

### How it works

A file is dragged on the file input, filling the input with the absolute path of the file. When submitted an output path is created from the given path by keep the original path and prefixing the file name with `removed-`.

The given path is checked for existence and the output path is checked for non existence, as to not accidentally overwrite a file. When any of these checks fail an error toast is displayed with the appropriate message.

The file is then sent through the API, when an error (eg: incorrect API key, no credits etc..) happens the message is displayed in an error toast. If all is good, the image will be saved to disk at the output path location and the image will be displayed in Raycast.

## Type of change

- New extension

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @maantje 👋 

Seems to be quite handy for removing backgrounds in a snap 😄 🔥 

The first thing I noticed was when adding a wrong API key and submitting an image, then this screen loads:

CuhxVD7RbgI3sZnS

Could you add some gracefully handling of that also?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by maantje** [comment]
Hi @pernielsentikaer, 

Thank you for reviewing and pointing out the bug. I pushed a commit to address this problem, and also fixed another bug with the crop checkbox not working correctly.

File: extensions/remove.bg/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "remove-bg",
+  "title": "Remove BG",
```
Comment (pernielsentikaer) [inline extensions/remove.bg/package.json:4]:
```suggestion
  "name": "remove-background",
  "title": "Remove Background",
```

File: extensions/remove.bg/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "description": "Removes the background from an image with remove.bg",
+  "icon": "command-icon.png",
+  "author": "maantje",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "remove.bg",
+      "subtitle": "Remove the background from an image",
```
Comment (pernielsentikaer) [inline extensions/remove.bg/package.json:13]:
```suggestion
      "title": "Remove Background From Image",
      "subtitle": "remove.bg",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @maantje 👋 

Thanks for the update 🔥 

I left a few things for you to address, also, you should rename the folder to `remove-background` instead, it makes more sense.

If I just submit without doing anything, I get this screen:
qJvC4QauXZab7VgN

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by maantje** [comment]
Hi @pernielsentikaer,

Thank you for re-reviewing, I addressed the points you brought up. Should I remove the showcase gif from the repo and link to it from somewhere else? 

I feel like the 7.05 MB is quite large.

**Comment by pernielsentikaer** [comment]
Hi @maantje 👋 

I think we should remove the showcase yes, but we should replace it with a step-by-step guide on how to use it in the `README` 😄 

When it's done, I think we're ready to merge 💯

**Pull request merged**